big_tests/tests/rdbms_SUITE.erl

```diff
@@ -601,38 +609,38 @@ sql_prepare_upsert(_Config, Name, Table, Insert, Update, Unique, Incr) ->
     escalus_ejabberd:rpc(rdbms_queries, prepare_upsert, [host_type(), Name, Table, Insert, Update, Unique, Incr]).
 
 sql_execute(_Config, Name, Parameters) ->
-    slow_rpc(mongoose_rdbms, execute, [host_type(), Name, Parameters]).
+    slow_rpc(mongoose_rdbms, execute, [host_type(), tag(), Name, Parameters]).
 
 sql_execute_cast(_Config, Name, Parameters) ->
-    slow_rpc(mongoose_rdbms, execute_cast, [host_type(), Name, Parameters]).
+    slow_rpc(mongoose_rdbms, execute_cast, [host_type(), tag(), Name, Parameters]).
```

Comment:
but it doesn't test version without a tag now

Reply:
Every other test suite is still testing the old way indirectly so I thought no need to duplicate test time here 🤔